.eslintrc.json

```diff
@@ -32,6 +32,14 @@
       "rules": {
         "@typescript-eslint/no-explicit-any": "off"
       }
+    },
+    {
+      "files": ["*.js", "*.mjs"],
+      "extends": ["eslint:recommended"],
+      "rules": {
+        "@typescript-eslint/no-explicit-any": "off",
```

Comment:
Is this so we don't need to fix them in this PR or is this the recommendation?
Perhaps add a comment?

Reply:
Added comment about intent.